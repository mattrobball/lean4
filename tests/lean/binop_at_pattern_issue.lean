open BitVec
def f4 (v : BitVec 32) : Nat :=
  match v with
  | 10#20 ++ 0#12  => 0 -- Should be rejected since `++` does not have `[match_pattern]`
  | _ => 1
