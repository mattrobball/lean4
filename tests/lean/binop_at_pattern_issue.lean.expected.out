binop_at_pattern_issue.lean:4:4-4:17: error: invalid pattern, constructor or constant marked with '[match_pattern]' expected
